modules/cluster-logging-kibana-limits.adoc

```diff
@@ -2,7 +2,6 @@
 //
 // * observability/logging/cluster-logging-visualizer.adoc
 
-:_mod-docs-content-type: PROCEDURE
```

Comment:
According to https://access.redhat.com/support/policy/updates/openshift_operators. Elasticsearch-opertaor in 4.16,4.17 is for trace only. I think we needn't remove all section as elasticsearch and kibana as interal store.